### PR TITLE
react native handler correction

### PR DESCRIPTION
**Pull request by ajaybhatt17**

### CODE REVIEW

File: lib/handlers/ReactNative.js [hunk 1]
```diff
@@ -56,6 +56,15 @@ class Handler extends EnhancedEventEmitter
...
 		});
+
+		this._pc.onnegotiationneeded = () =>
+		{
+			if (isOffer)
+			{
+				this.createOffer();
+			}
+		};
+
```
Comment (ibc) [inline lib/handlers/ReactNative.js]: What is this `createOffer()` for?

File: lib/handlers/ReactNative.js [hunk 2]
```diff
@@ -66,13 +75,26 @@ class Handler extends EnhancedEventEmitter
...
+		this._pc.createOffer().then((desc) =>
+		{
+			return this._pc.setLocalDescription(desc);
+		})
+			.catch((err) =>
+			{
+				logger.error(err);
+			});
+	}
+
```
Comment (ibc) [inline lib/handlers/ReactNative.js]: I don't understand the purpose of this.

**Comment by ibc** [comment]
I don't understand the purpose of this PR. To be clear: I have `mediasoup-client` working fine in `react-native` by using a modified version of `react-native-webrtc` (just the commit I recently shown in the mediasoup mailing list).

Not sure the rationale behind the `onnegotiationeeded`.

Also, the PR does not follow the `mediasoup-client` code syntax. But, before that, I need to understand why this PR is needed at all and what it is supposed to provide or fix.

**Comment by ajaybhatt17** [comment]
The changes is for handling remote peers video which is not showing. 

Sorry if i missed, which commit you are refering ... let me try that one also.

And above two snippet code (onnegotiationeeded and createOffer) are refer from https://github.com/oney/RCTWebRTCDemo/blob/master/main.js

**Comment by ibc** [comment]
> The changes is for handling remote peers video which is not showing.

Sorry, that's not needed at all and, in fact, breaks mediasoup-client usage.


> Sorry if i missed, which commit you are refering ... let me try that one also.

This: https://groups.google.com/d/msg/mediasoup/107-9cb98zE/NHCj6CyBCAAJ


> And above two snippet code (onnegotiationeeded and createOffer) are refer from https://github.com/oney/RCTWebRTCDemo/blob/master/main.js

Just because a `react-native-webrtc` demo project uses that it does not mean that every WebRTC usage with `react-native-webrtc` requires that. `mediasoup-client` does not work that way at all.

Sorry, I cannot accept this PR.

**Comment by ajaybhatt17** [comment]
Yes you are right.. tried the mentioned code in react-native-webrtc, it is working. Sorry for your trouble.

Thanks

**Comment by ibc** [comment]
Great :)

**Pull request closed without merging**